libdebug/interfaces/debugging_interface.py

```diff
@@ -94,6 +94,10 @@ def finish(self: DebuggingInterface, thread: ThreadContext, heuristic: str) -> N
             heuristic (str, optional): The heuristic to use. Defaults to "backtrace".
         """
 
+    def next(self: DebuggingInterface, thread: ThreadContext) -> None:
```

Comment:
same comment of other files

Reply:
Resolved in [fb11af7](https://github.com/libdebug/libdebug/pull/80/commits/fb11af7cf439a12391ae1831aee03fcd2b39b45f)